systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/NetworkPolicyUtils.java

```diff
@@ -165,14 +135,14 @@ public static void allowNetworkPolicyAllIngressForMatchingLabel(String namespace
                 .editSpec()
                     // keeping ingress empty to allow all connections
                     .addNewIngress()
-                    .endIngress()
-                    .withNewPodSelector()
+                        .endIngress()
+                            .withNewPodSelector()
```

Comment:
nit indents

Reply:
Yeah I have change for this locally, thanks.